wsd/DocumentBroker.cpp

```diff
@@ -1674,6 +1674,13 @@ void DocumentBroker::alertAllUsers(const std::string& msg)
     }
 }
 
+void DocumentBroker::setKitLogLevel(const std::string& level)
+{
+    assertCorrectThread();
+    std::unique_lock<std::mutex> lock(_mutex);
```

Comment:
If it's called form the "correct thread" then we don't need the lock. See other cases where we access _childProcess and also the (few) cases where the lock is used.

Reply:
I removed lock.